modules/tinymce/src/core/test/ts/browser/undo/UndoKeyboardShortcutsTest.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.undo.UndoKeyboardShortcutTest', () => {
+  const platform = PlatformDetection.detect();
+
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce'
+  }, [], true);
+
+  const selectAll = (editor: Editor) => {
+    editor.execCommand('SelectAll');
+  };
+
+  const undoKeyStroke = (editor: Editor) => {
+    TinyContentActions.keystroke(editor, 'Z'.charCodeAt(0), platform.os.isMacOS() ? { metaKey: true } : { ctrl: true });
+  };
+
+  const redoKeyStroke = (editor: Editor) => {
```

Comment:
`KeyStroke` should be `Keystroke` in these 2 function names.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/4cb9179c81f47841bf2fedc0e160071fd5f68cb5